src/test/java/com/project/parkinglot/service/parking_area/impl/ParkingAreaGetServiceImplTest.java

```diff
@@ -123,5 +126,51 @@ void givenNonExistParkingArea_whenGetParkingAreaByName_thenThrowsParkingAreaNotF
         Mockito.verify(parkingAreaRepository, Mockito.times(1)).findByName(Mockito.anyString());
 
     }
+    @Test
+    void givenValidGetDailyIncomeParameter_WhenGivenDailyIncome_ThenReturnParkingAreaIncomeResponse(){
+
+        // Given
+        String mockParkingAreaId = UUID.randomUUID().toString();
+
+        ParkingAreaEntity mockParkingAreaEntity = new ParkingAreaEntityBuilder()
+                .withValidFields()
+                .withId(mockParkingAreaId)
+                .build();
+
+        ParkingAreaIncomeResponse mockParkingAreaIncomeResponse = new ParkingAreaIncomeResponseBuilder()
+                .withValidFields()
+                .withName(mockParkingAreaEntity.getName())
+                .build();
+
+        LocalDate mockDate = generateRandomDate();
+
+
+
+
+        // When
+        Mockito.when(parkingAreaRepository.findById(mockParkingAreaId)).thenReturn(Optional.of(mockParkingAreaEntity));
+
+        Mockito.when(parkingAreaRepository.calculateDailyIncome(mockDate,mockParkingAreaId)).thenReturn(Optional.of(BigDecimal.valueOf(Mockito.anyLong())));
+
+        // Then
+        ParkingAreaIncomeResponse checkParkingAreaIncomeResponse = parkingAreaService.getDailyIncome(mockDate,mockParkingAreaId);
+        Assertions.assertEquals(checkParkingAreaIncomeResponse.getName(),mockParkingAreaIncomeResponse.getName());
+
+        //Verify
+        Mockito.verify(parkingAreaRepository,Mockito.times(1)).findById(mockParkingAreaId);
+        Mockito.verify(parkingAreaRepository,Mockito.times(1)).calculateDailyIncome(mockDate,mockParkingAreaId);
+
+
+    }
+
+
+
+
+    private LocalDate generateRandomDate(){
```

Comment:
Remove generateRandomDate from this test class.

Create Util class to call it.